src/Microsoft.AspNet.Razor/Parser/CSharpCodeParser.Statements.cs

```diff
@@ -254,7 +257,7 @@ private void AfterTryClause()
             {
                 Accept(ws);
```

Comment:
Unrelated but since you are touching this, change to `var` here https://github.com/aspnet/Razor/pull/421/files#diff-f4f601d3f9c913c93d41ea331034c64fR253